### DISCUSSION TITLE
Make civilization on europa destroyable.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5971] [Ideas and suggestions]
TLDR; Let us play as rebels/cultists and give us the ability to destroy/infect stations.

Would be awesome if we could roleplay more as the badguys in Barotrauma, such as the husk cult, or jovian terrorists, and be able to actually destroy/infect outposts, removing them from the map. Of course, this would not be without consequences, when/if submarine AI is implemented, the coalition would come after you with subs. Additionally, most outposts except for those aligned with your faction would turn against you, or might even let a host of coalition marines into your sub.

**Comment by Veritius** [comment]
A better approach to what you're trying to say is "when I blow up a station, make it meaningful."
You can already destroy entire stations, but once you leave, it's like it never happened (apart from reputation effects)
Additionally, stations doing more to seem like something other than what they are: safezones where you can trade.

Afaik, you're always a crew of pretty neutral guys who just want to continue living.
Right now, you can install a mod (I think it was Thalassophobia) that allows you to change your starting conditions such as rep with factions. It's not really that impactful on gameplay because of the limitations of the content package system, but it definitely adds something to immersion.
